tools/utils/fiber-helpers.ts

```diff
@@ -89,34 +87,35 @@ _.extend(exports.EnvironmentVariable.prototype, {
     return () => {
       currentValues[this.slot] = saved;
     };
-  },
+  }
 
-  withValue(value, func) {
+  withValue(value: any, func: Function) {
     const reset = this.set(value);
     try {
       return func();
     } finally {
       reset();
     }
   }
-});
+}
 
 // This is like Meteor.bindEnvironment.
 // Experimentally, we are NOT including onException or _this in this version.
-exports.bindEnvironment = function (func) {
-  exports.nodeCodeMustBeInFiber();
+export function bindEnvironment(func: Function) {
+  nodeCodeMustBeInFiber();
 
-  var boundValues = _.clone(Fiber.current._meteorDynamics || {});
+  const boundValues = { ...(Fiber.current._meteorDynamics || {}) };
 
-  return function (...args) {
-    var self = this;
+  return function (...args: any[]) {
+    //@ts-ignore
+    const self = this;
 
-    var runWithEnvironment = function () {
-      var savedValues = Fiber.current._meteorDynamics;
+    const runWithEnvironment = () => {
```

Comment:
Since you’ve turned this into an arrow function, I think you should be able to safely remove the `self` variable and the `//@ts-ignore`.